docs/book/v1/how-tos/create-pages.md

```diff
@@ -1,17 +1,23 @@
 # Creating pages
 
-## The `action` function
+## Register the page route
 
-The first step is to add the new pages in `src/Page/src/Controller/PageController.php`.
-This means adding an `Action` function for each page, as seen below.
+Open `config/autoload/local.php` and locate the **routes** key.
+Under the **page** key you will find an associative array with a couple of already existing routes.
+You create a new route by appending a new line to that array:
 
 ```php
-    public function examplePageAction(): ResponseInterface
-    {
-        return new HtmlResponse(
-            $this->template->render('page::example-template')
-        );
-    }
+'example-page' => 'example-template',
+```
+
+> The array key is the page URI, the array value is the template name used by that page.
```

Comment:
```suggestion
> The array key is the page slug, the array value is the template name used by that page.
```
Since my review on the `routing.md` file, calling this slug instead of URI seems to make more sense because it refers only to the `about` part, not the whole path `/page/about`.